### DISCUSSION TITLE
drop animation issue (going back to the place from where I dragged it)

### DISCUSSION THREAD

**Discussion by piszczu4** [clauderic/dnd-kit#1558] [Q&A]
I want to have 4 draggable symbols : +, -, :, * and many slots where I could drop these symbols. The issue is that I can't make the drop animation working. When I drop item it should animate nicely to the slot where I dropped it, but instead it is animating back to the place from where I dragged it. Here is my code: https://stackblitz.com/edit/vitejs-vite-hivtj4mm?file=package.json,src%2Fdroppable-container.tsx,src%2FApp.tsx,src%2Fdraggable-item.tsx,src%2Fitem.tsx,src%2Fmain.tsx

Any ideas how to fix it?

**Comment by Dennis-N8** [comment]
Same question here.
